conans/client/conf/config_installer.py

```diff
@@ -130,7 +130,7 @@ def _process_folder(config, folder, cache, output):
         folder = os.path.join(folder, config.source_folder)
     for root, dirs, files in walk(folder):
         dirs[:] = [d for d in dirs if d != ".git"]
-        if ".git" in root:
+        if os.path.join(".git", "") in root:
```

Comment:
I'm not sure if this will make a difference in the end, the line above: `dirs[:] = [d for d in dirs if d != ".git"]`
Is already filtering the _.git_ folder.
Also, the new comparison will compare if `.git/` is in root that will have the value `my/root/path/git` so it will always be false because of the forward slash join is going to add in the end.
I don't know if I'm missing something, but is this comparison needed in the end?